documentation/shared/snip-intro-operators.adoc

```diff
@@ -0,0 +1,4 @@
+//standard operator intro text
+Operators are a method of packaging, deploying, and managing a Kubernetes application.
+{ProductName} Operators extend Kubernetes functionality, automating common and complex tasks related to a Kafka deployment.
```

Comment:
Could give an example.

Reply:
Not sure this snippet is the right place, as we may use it in more than one situation.